cypress/features/regression/designSystem/grid.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Design System Grid component
```

Comment:
Do we still need these file with tests in DS since we have visual tests now ?

Reply:
I think so, it's a new functionality

Reply:
removed